### PR TITLE
Add long short equity strategy script to examples

### PR DESCRIPTION
**Pull request by andrewkim316**

### CODE REVIEW

File: examples/long-short.js [hunk 1]
```diff
@@ -0,0 +1,423 @@
+const API_KEY = 'PK2QQWOEG6ALQ0ADZV3H';
+const API_SECRET = 'f393ILtkPrxxnFgESMzSeDnDkFiDB9zXUZcJLjm0';
```
Comment (umitanuki) [inline examples/long-short.js]: Is this the real key? Regardless, it should fail clearly unless the reader sets the right key.

File: examples/long-short.js [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+        qty: quantity,
+        side: side,
+        type: 'market',
+        time_in_force: 'day',
+      }).then(() => {
+        console.log("Market order of " + quantity + " " + stock + ", " + side);
+      });
+    }
+    else {
+      console.log("Quantity is 0");
```
Comment (umitanuki) [inline examples/long-short.js]: maybe it's helpful to log symbol as well.

File: examples/mean-reversion.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+        setTimeout(() => {
+          // Run script again after market close for next trading day
+          this.run();
+        },60000*15);
+      }
+    },60000);
+  }
+
+  // Spin until the market is open
+  awaitMarketOpen(){
```
Comment (umitanuki) [inline examples/mean-reversion.js]: Should this be shared among algos?

File: examples/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+## Mean Reversion
+
+This trading algorithm bases its strategy on a mean reversion theory, which essentially in the long run, the stock price will correct to the mean.  This means we'd want to execute trades when the stock price is below the running average, as the theory states that the stock price will eventually rise to the mean.  The algorithm does this by taking the 20 minute running average stock price of a given stock (in this case "AAPL") and longs or sells based on the average.  After every minute, the algorithm will re-evaluate the mean and see if adjustments to the position need to be made.  For more information on this strategy, you can read [this link](https://medium.com/automation-generation/a-simple-mean-reversion-stock-trading-script-in-c-fdd3d147af95) detailing a mean reversion strategy in C#.
```
Comment (ttt733) [inline examples/README.md]: Please make the same changes to this that you made to the README in Go.

**Pull request merged**